5-app-infra/README.md

```diff
@@ -50,34 +44,728 @@ Hub and Spoke network model. It also sets up the global DNS hub</td>
+
+   ```bash
+   git push --set-upstream origin plan
+   ```
+
+1. Merge changes to shared. Because this is a [named environment branch](../docs/FAQ.md#what-is-a-named-branch),
+   pushing to this branch triggers both _terraform plan_ and _terraform apply_. Review the apply output in your Cloud Build project https://console.cloud.google.com/cloud-build/builds;region=DEFAULT_REGION?project=YOUR_INFRA_PIPELINE_PROJECT_ID
+
+   ```bash
+   git checkout -b shared
+   git push origin shared
+   ```
+
+1. `cd` out of the `bu3-service-catalog` repository.
+   
+   ```bash
+   cd ..
+   ```
+
+1. Navigate to the project that was output from `${ARTIFACT_PROJECT_ID}` in Google's Cloud Console to view the first run of images being built.
+
+1. Grab the Artifact Project ID
+   
+   ```bash
+   export ARTIFACT_PROJECT_ID=$(terraform -chdir="gcp-projects/business_unit_3/shared" output -raw common_artifacts_project_id)
+   echo ${ARTIFACT_PROJECT_ID}
+   ```
+
+1. Clone the freshly minted Cloud Source Repository that was created for this project.
+   
+   ```bash
+   gcloud source repos clone publish-artifacts --project=${ARTIFACT_PROJECT_ID}
+   ```
+
+1. Enter the repo folder and copy over the artifact files from `5-app-infra/source_repos/artifact-publish` folder.
+   
+   ```bash
+   cd publish-artifacts
+   git checkout -b main
+
+   git commit -m "Initialize Repository" --allow-empty
+   cp -RT ../terraform-google-enterprise-genai/5-app-infra/source_repos/artifact-publish/ .
+   ```
+
+1. Commit changes and push your main branch to the new repo.
+
+   ```bash
+   git add .
+   git commit -m 'Build Images'
+
+   git push --set-upstream origin main
+   ```
+
+1. `cd` out of the `publish-artifacts` repository.
+   
+   ```bash
+   cd ..
+   ```
+
+1. Grab the Service Catalogs ID
+  
+   ```bash
+   export SERVICE_CATALOG_PROJECT_ID=$(terraform -chdir="gcp-projects/business_unit_3/shared" output -raw service_catalog_project_id)
+   echo ${SERVICE_CATALOG_PROJECT_ID}
+   ```
+
+1. Clone the freshly minted Cloud Source Repository that was created for this project.
+   
+   ```bash
+   gcloud source repos clone service-catalog --project=${SERVICE_CATALOG_PROJECT_ID}
+   ```
+
+1. Enter the repo folder and copy over the service catalogs files from `5-app-infra/source_repos/service-catalog` folder.
+   
+   ```bash
+   cd service-catalog/
+   cp -RT ../terraform-google-enterprise-genai/5-app-infra/source_repos/service-catalog/ .
+   ```
+
+1. Commit changes and push main branch to the new repo.
+   
+   ```bash
+   git add .
+   git commit -m 'Initialize Service Catalog Build Repo'
+
+   git push --set-upstream origin main
+   ```
+
+1. `cd` out of the `service_catalog` repository.
+   
+   ```bash
+   cd ..
+   ```
+
+1. Navigate to the project that was output from `${ARTIFACT_PROJECT_ID}` in Google's Cloud Console to view the first run of images being built.
+
+#### VPC-SC
```

Comment:
Remove VPC-SC and INGRESS/EGRESS policies

Move it to the 6-ml-pipeline

Also note that on repo root contains [6-machine-learning](https://github.com/GoogleCloudPlatform/terraform-google-enterprise-genai/tree/a39cce24cc129f6e79c2b1c7953a38a5c91f17d5/6-machine-learning) and [6-ml-pipeline/dev](https://github.com/GoogleCloudPlatform/terraform-google-enterprise-genai/tree/a39cce24cc129f6e79c2b1c7953a38a5c91f17d5/6-ml-pipeline/dev)

both should be merged into one